src/appleseed-max-impl/appleseedinteractive/interactivesession.h

```diff
@@ -58,10 +62,12 @@ class InteractiveSession
   private:
     void render_thread();
 
+    RendProgressCallback*                           m_progress_cb;
```

Comment:
It is *very* important that the order of data member declarations and the order of data member initialization match. This doesn't appear to be the case right now.